iex_trading/anaconda-project.yml

```diff
@@ -3,36 +3,35 @@
 name: IEX trading
 description: Dashboard visualizing stock trades on the IEX exchange
 maintainers:
-  - jlstevens
+- jlstevens
 labels:
-  - datashader
-  - panel
-  - holoviews
-  - dev
-  - channel_pyviz-dev
-  
+- datashader
+- panel
+- holoviews
+- dev
+- channel_pyviz-dev
```

Comment:
@jlstevens, the dev label can now be removed, right? And the dev channel below, and bump hv up to 1.13? If so please do so here.